inngest/_internal/server_lib/registration_test.py

```diff
@@ -42,6 +42,7 @@ def test_serialization() -> None:
             period=datetime.timedelta(seconds=60),
         ),
         id="foo",
+        idempotency="foo",
```

Comment:
**suggestion**: For the sake of posterity for future maintainers or contributors, can we change this to `event.data.foo`? That would make it feel also more obvious as how an expression might actually be written ([docs](https://www.inngest.com/docs/guides/handling-idempotency#example)).
```suggestion
        idempotency="foo",
```